llvm/test/CodeGen/PowerPC/mma-acc-spill-p11.ll

```diff
@@ -0,0 +1,122 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; ; This test is a copy of mma-acc-spill.ll except that it uses mcpu=pwr11.
+; RUN: llc -verify-machineinstrs -mtriple=powerpc64le-unknown-linux-gnu \
+; RUN:   -mcpu=pwr11 -ppc-asm-full-reg-names -disable-auto-paired-vec-st=false \
```

Comment:
I don't quite follow why this test got versioned. If I understand correctly, the codegen should be identical (at least for now), so I think we should just add the extra run lines to the existing test if anything.